functions/index.js

```diff
@@ -0,0 +1,25 @@
+const functions = require("firebase-functions");
+const fs = require("fs");
+exports.preRender = functions.https.onRequest((request, response) => {
+  const path = request.path ? request.path.split("/") : request.path;
+  let index = fs.readFileSync("./web/index.html").toString();
+  const setMetas = (title, description) => {
+    index = index.replace("INJECTED_TITLE", title);
+    index = index.replace("INJECTED_DESCRIPTION", description);
+  };
+  switch (path[1]) {
+    case "faq": {
+      setMetas("FAQ", "Frequently Asked Questions");
+    } break;
+    case "privacy": {
+      setMetas("Privacy", "Privacy Politics");
+    } break;
+    case "terms": {
+      setMetas("Terms", "Terms and Conditions");
+    } break;
+    default: {
+      setMetas("TestA", "Discover spontaneous activities");
```

Comment:
"TestA" I don't think this should go to production

Reply:
This issue has been resolve.